knora/dsplib/models/resource.py

```diff
@@ -417,11 +407,27 @@ def get_resclass_type(self, prefixedresclass: str) -> Type:
             'knora-api:LinkValue': LinkValue,
         }
         for propname, has_property in resclass.has_properties.items():
-            if propname == "knora-api:isPartOf":
+            if any([
+                propname == "knora-api:isAnnotationOf",
+                propname == "knora-api:isRegionOf",
+                propname == "knora-api:isPartOf",
+                propname == "knora-api:hasLinkTo"
+            ]):
```

Comment:
imo slightly more elegant (and shorter to write) is `if propname in {"knora-api:isAnnotationOf", ...}`

Reply:
I will keep that in mind for the future, thanks for the hint!